usr/sbin/regionsrv-enabler-azure

```diff
@@ -12,6 +12,34 @@
 # Lesser General Public License for more details.
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library.
+
+"""
+Check the different states of the license type:
+    START   LICENSE CHANGED TO     CACHE   ACTION
+    BYOS-1  SLES                   N/A     Register
```

Comment:
They way I understand this this would read as:

"""
An unregistered BYOS instance that gets moved to PAYG via AHB and has no cahce file needs to get registered to the update infrastructure
"""

If this is correct I suggest we express this as

`
BYOS-1   PAYG      No  Register - Update Infra
`

Reply:
Yes, that is the intend

I will update the wording